distributions/Dist_kjm538.py

```diff
@@ -14,10 +14,12 @@ def pdf(self, x):
 
 	def mean(self):
 		"""This is the mean of the PDF"""
-		return (1./23.)*((36*np.log(36))-35)
+		return (1./23.)*((self.x_max**2 * np.log(self.x_max**2))
+		-(self.x_max**2)+1)
 
 	def std(self):
 		"""This is the standard deviation of the pdf"""
-		return np.sqrt((4./23.)*((-24+(50110707.*(np.log(6)-1))/500000.)
-		+(72*np.log(6))-(36783./500.)*(np.log(36)-1))+(5780027./2250000.))
-
+		return np.sqrt((4./23.)*((-((self.x_max**3)/9.)+
+		(50110707.*(np.log(self.x_max)-1))/500000.)
+		+(((self.x_max**3)/3.)*np.log(self.x_max))-(36783./500.)*
+		(np.log(self.x_max**2)-1))+(5780027./2250000.))
```

Comment:
what is the source of these crazy looking numbers? You can write comments in the code too.